[2020-01-17 14:22:01,350] {{taskinstance.py:655}} INFO - Dependencies all met for <TaskInstance: getGmailAttachments.hello_task 2020-01-17T14:21:56.043503+00:00 [queued]>
[2020-01-17 14:22:01,361] {{taskinstance.py:655}} INFO - Dependencies all met for <TaskInstance: getGmailAttachments.hello_task 2020-01-17T14:21:56.043503+00:00 [queued]>
[2020-01-17 14:22:01,361] {{taskinstance.py:866}} INFO - 
--------------------------------------------------------------------------------
[2020-01-17 14:22:01,362] {{taskinstance.py:867}} INFO - Starting attempt 1 of 1
[2020-01-17 14:22:01,362] {{taskinstance.py:868}} INFO - 
--------------------------------------------------------------------------------
[2020-01-17 14:22:01,387] {{taskinstance.py:887}} INFO - Executing <Task(PythonOperator): hello_task> on 2020-01-17T14:21:56.043503+00:00
[2020-01-17 14:22:01,392] {{standard_task_runner.py:52}} INFO - Started process 24247 to run task
[2020-01-17 14:22:01,479] {{logging_mixin.py:112}} INFO - [2020-01-17 14:22:01,475] {{dagbag.py:403}} INFO - Filling up the DagBag from /usr/local/airflow/dags/email.py
[2020-01-17 14:22:01,500] {{logging_mixin.py:112}} INFO - Running %s on host %s <TaskInstance: getGmailAttachments.hello_task 2020-01-17T14:21:56.043503+00:00 [running]> 7f68746faaa4
[2020-01-17 14:22:01,516] {{python_operator.py:105}} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=getGmailAttachments
AIRFLOW_CTX_TASK_ID=hello_task
AIRFLOW_CTX_EXECUTION_DATE=2020-01-17T14:21:56.043503+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2020-01-17T14:21:56.043503+00:00
[2020-01-17 14:22:01,517] {{logging_mixin.py:112}} WARNING - /usr/local/lib/python3.7/site-packages/oauth2client/_helpers.py:255: UserWarning: Cannot access /usr/local/airflow/dags/token.json: No such file or directory
  warnings.warn(_MISSING_FILE_MESSAGE.format(filename))
[2020-01-17 14:22:01,519] {{logging_mixin.py:112}} WARNING - usage: airflow [--auth_host_name AUTH_HOST_NAME] [--noauth_local_webserver]
               [--auth_host_port [AUTH_HOST_PORT [AUTH_HOST_PORT ...]]]
               [--logging_level {DEBUG,INFO,WARNING,ERROR,CRITICAL}]
[2020-01-17 14:22:01,520] {{logging_mixin.py:112}} WARNING - airflow: error: unrecognized arguments: run getGmailAttachments hello_task 2020-01-17T14:21:56.043503+00:00 --local --pool default_pool -sd /usr/local/airflow/dags/email.py
[2020-01-17 14:22:11,326] {{logging_mixin.py:112}} INFO - [2020-01-17 14:22:11,326] {{local_task_job.py:103}} INFO - Task exited with return code 0
